tools/bcp47.py

```diff
@@ -262,7 +265,6 @@
     "sa" : "sanskrit",
     "sa-Deva" : "sanskrit",
     "sa-Beng" : "sanskrit",
-    "sa-Deva" : "sanskrit",
```

Comment:
dito

Reply:
The key `sa-Deva` is already defined with the same value two lines above the deleted line, so this line can be safely removed. See the response to the other comment as well (https://github.com/reutenauer/polyglossia/pull/649#discussion_r1675718376).

Reply:
thanks